astrocut/cutouts.py

```diff
@@ -271,8 +274,15 @@ def fits_cut(input_files, coordinates, cutout_size, correct_wcs=False, extension
         if verbose:
             print("\nCutting out {}".format(in_fle))
 
+        u_fsspec = None
+        fsspec_kw = None
+
+        if "s3://" or "gs://" in in_fle:
```

Comment:
Need to remove `gs://`, the current plan is to use AWS S3 for cloud-storage of STScI data.